src/main/java/roomescape/presentation/api/reservation/AdminReservationController.java

```diff
@@ -0,0 +1,48 @@
+package roomescape.presentation.api.reservation;
+
+import jakarta.validation.Valid;
+import java.time.LocalDate;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.application.reservation.ReservationService;
+import roomescape.application.reservation.dto.ReservationResult;
+import roomescape.application.reservation.dto.ReservationSearchParam;
+import roomescape.presentation.api.reservation.request.CreateAdminReservationRequest;
+import roomescape.presentation.api.reservation.response.ReservationResponse;
+
+@RestController
+@RequestMapping("/admin/reservations")
+public class AdminReservationController {
+
+    private final ReservationService reservationService;
+
+    public AdminReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
+
+    @PostMapping
+    public ResponseEntity<ReservationResponse> createReservation(
+            @Valid @RequestBody CreateAdminReservationRequest createAdminReservationRequest) {
+        Long reservationId = reservationService.create(createAdminReservationRequest.toServiceParam());
+        ReservationResult reservationResult = reservationService.findById(reservationId);
+        return ResponseEntity.status(HttpStatus.CREATED).body(ReservationResponse.from(reservationResult));
+    }
+
+    @GetMapping
+    public ResponseEntity<?> findAll(@RequestParam("themeId") Long themeId, @RequestParam("memberId") Long memberId,
```

Comment:
```suggestion
    public ResponseEntity<List<ReservationResponse>> findAll(@RequestParam("themeId") Long themeId, @RequestParam("memberId") Long memberId,
```
명시적인 타입을 지정해줘도 좋을 것 같아요~